src/shared/components/test/Buttons-test.js

```diff
@@ -0,0 +1,343 @@
+    });
+
+    it('renders children', () => {
+      const wrapper = createComponent();
+
+      assert.equal(wrapper.text(), 'Do this');
+    });
+
+    it('uses a default className', () => {
+      const wrapper = createComponent();
+
+      assert.isTrue(wrapper.find('button').hasClass('LabeledButton'));
+    });
+
+    it('renders a primary variant', () => {
+      const wrapper = createComponent({ variant: 'primary' });
+
+      assert.isTrue(wrapper.find('button').hasClass('LabeledButton--primary'));
+    });
+
+    it('overrides className when provided', () => {
+      const wrapper = createComponent({
+        className: 'CustomClassName',
+        variant: 'primary',
+      });
+
+      assert.isTrue(wrapper.find('button').hasClass('CustomClassName'));
+      assert.isTrue(
+        wrapper.find('button').hasClass('CustomClassName--primary')
+      );
+      assert.isFalse(wrapper.find('button').hasClass('IconButton'));
+    });
+
+    [
+      {
+        propName: 'expanded',
+        propValue: true,
+        attributeName: 'aria-expanded',
+        attributeValue: 'true',
+      },
+      {
+        propName: 'pressed',
+        propValue: true,
+        attributeName: 'aria-pressed',
+        attributeValue: 'true',
+      },
+      {
+        propName: 'title',
+        propValue: 'Click here',
+        attributeName: 'aria-label',
+        attributeValue: 'Click here',
+      },
+      {
+        propName: 'disabled',
+        propValue: true,
+        attributeName: 'disabled',
+        attributeValue: '',
+      },
+    ].forEach(testCase => {
+      it('sets attributes on the button element', () => {
+        const wrapper = createComponent({
+          [testCase.propName]: testCase.propValue,
+        });
+
+        const element = wrapper.find('button').getDOMNode();
+
+        assert.equal(
+          element.getAttribute(testCase.attributeName),
+          testCase.attributeValue
+        );
+      });
+    });
+  });
+
+  describe('LinkButton', () => {
+    let fakeOnClick;
+
+    function createComponent(props = {}) {
+      return mount(
+        <LinkButton title="My Action" onClick={fakeOnClick} {...props}>
+          Do this
+        </LinkButton>
+      );
+    }
+
+    beforeEach(() => {
+      fakeOnClick = sinon.stub();
+      $imports.$mock(mockImportedComponents());
+    });
+
+    afterEach(() => {
+      $imports.$restore();
+    });
+
+    it('invokes callback on click', () => {
+      const onClick = sinon.stub();
+      const wrapper = createComponent({ onClick });
+
+      wrapper.find('button').simulate('click');
+      assert.calledOnce(onClick);
+    });
+
+    it('renders children', () => {
+      const wrapper = createComponent();
+
+      assert.equal(wrapper.text(), 'Do this');
+    });
+
+    it('uses a default className', () => {
+      const wrapper = createComponent();
+
+      assert.isTrue(wrapper.find('button').hasClass('LinkButton'));
+    });
+
+    it('renders a primary variant', () => {
```

Comment:
```suggestion
    it('renders a primary `variant`', () => {
```

I kinda think we add ticks to prop names in test descriptions